docs/design/conan.md

```diff
@@ -0,0 +1,373 @@
+# Adding Conan Support to Hermeto
+
+## Background
+
+[Conan][] is one of many package managers for C++. At the moment of writing
+it is not abandoned and supports quite a few C++ libraries. Support for this
+PM was  inquired about by users. Conan provides a lot of functionality out of the
+box: all major platforms are supported, integration with multiple build
+systems exists, can handle both pre-compiled binaries and build from source,
+has an official [central repository][] for hosting community provided packages
+as well as an option for hosting [private repositories][]. Conan is written in
+Python and could be used both as a stand-alone tool and as a library (however
+Python APIs are [considered experimental][]). Conan allows to use Python as
+a dependencies specification language as well which creates ample opportunities
+for arbitrary code execution during both collection and build time.
+
+Conan is [available via PyPI][] or [as rpm][].
+
+Technically Conan cannot download dependencies from anywhere except from a
+central repository, however it is possible to amend _conanfile.py_
+in such a way that it would download a package from GitHub and would expose it
+via a local repository.
+
+Apparently it has nothing to do with a certain Cimmerian.
```

Comment:
:rofl:

Reply:
Probably not something that needs to be mentioned in a design doc :).